Source/WarpX.cpp

```diff
@@ -853,6 +853,8 @@ WarpX::ReadParameters ()
         pp_warpx.query("do_pml_j_damping", do_pml_j_damping);
         pp_warpx.query("do_pml_in_domain", do_pml_in_domain);
         pp_warpx.query("do_similar_dm_pml", do_similar_dm_pml);
+        v_particle_pml = PhysConst::c;
+        pp_warpx.query("v_particle_pml", v_particle_pml);
```

Comment:
Could you add a few lines of code to check that `v_particle_pml` is not zero (so after the variable is queried from the inputs, to make sure that the user did not enter `warpx.v_particle_pml = 0.` by mistake and thus avoid divisions by zero at runtime)? You could use the functionality provided by `WARPX_ALWAYS_ASSERT_WITH_MESSAGE` and find examples of how this is used in WarpX by grepping it within the source code (using `git grep`).